README.md

```diff
@@ -1,36 +1,42 @@
 # stonks - CSC207 course project
 
+## Installation Instructions
+
+1. Clone the repository using any CLI with git installed using "git clone https://github.com/awesominat/stonks.git" (alternatively, you can download the repo)
+2. Install all necessary packages (OkHttp, json)
+3. Create an account over on [Finnhub](finnhub.io) and copy your API key
+4. Create a file called file.txt and insert the key and nothing else (no spaces)
```

Comment:
Mention the filepath to place file.txt in?

Reply:
good idea